feat: configurable journey planner api configurations

mortennordseth opened the pull request:
Because the journey planner API configurations differ for AtB, NFK, and FRAM we need to have them configurable. This makes that possible by using the org data.

Closes https://github.com/AtB-AS/kundevendt/issues/16416

adriansberg commented:
How is this configured for the app? In the bff?

mikaelbr reviewed (changes requested):
I think this should be part of org data instead. It isn't needed as a secret but just easily accessible information

mortennordseth commented:
I've moved the configuration to the org data and updated the values to match what is used in the app. 

> How is this configured for the app? In the bff?

This is configured in the app. Some are editable with preferences

adriansberg commented on orgs/schema-validation.json:
```diff
@@ -75,6 +75,27 @@
           ],
           "type": "object"
+        },
+        "journeyAPIConfigurations": {
```
```suggestion
        "journeyApiConfigurations": {
```

adriansberg commented on orgs/schema-validation.json:
```diff
@@ -75,6 +75,27 @@
+        "journeyAPIConfigurations": {
+          "additionalProperties": false,
+          "properties": {
+            "WAIT_RELUCTANCE": {
```
Do these need to be capitalised when they're not defined as constants, but as properties in an object?

adriansberg reviewed (approved):
Looks good, other than some nitpicky comments.
